Automattic/jetpack
# Connection: remove in-place in secondary flows

#### Changes proposed in this Pull Request:
Remove in-place from the following connection flows:
- User connection bar in jetpack Settings
- User connection iframe on Jetpack Dashboard

#### Jetpack product discussion
p1HpG7-bZE-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
1. Disconnect Jetpack, connect it in site-only mode.
2. Go to Jetpack Dashboard, click "Connect" in one of the "Security" or "Performance" cards. The connection screen should appear, saying "Unlock all the amazing features..."
3. Click "Connect your user account". You should get redirected to Calypso connection flow. No need to "Approve" the connection for now, you can just go back to the Dashboard.
4. Scroll down to the "Account connection" card, click "Connect your WordPress.com" account. Confirm that the connection screen appears again.
5. Finish the connection, confirm that it works correctly, then disconnect. Connect again in site-only mode.
6. Go to "Settings", click "Connect your WordPress.com account" under one of the greyed out cards.
7. The connection screen should appear, but this time instead of the generic title, it should say "Unlock [feature name] and more amazing features".
8. Click "Connect your user account". You should get redirected to the Calypso flow. Complete the connection, confirm that it went fine.

#### The connect screen:
<img width="1146" alt="Screen Shot 2021-08-27 at 1 56 10 PM" src="https://user-images.githubusercontent.com/1341249/131169362-d1c5c3de-6ab5-401d-bd85-d48fbdb4cb84.png">
